heron/tools/cli/src/python/response.py

```diff
@@ -33,14 +34,17 @@ class Status(object):
   Ok = 0
   InvocationError = 1
   HeronError = 100
```

Comment:
We should include a comment here with a reference to the java class where these constants are defined.